File: medplum-java/medplum-model/src/main/java/com/fhir/CoverageEligibilityResponse_Item.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.Boolean;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableCoverageEligibilityResponse_Item.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface CoverageEligibilityResponse_Item {
  Optional<CodeableConcept> term();

  Optional<CodeableConcept> category();

  Optional<Boolean> authorizationRequired();

  Optional<List<Extension>> extension();

  Optional<String> description();

  Optional<List<Extension>> modifierExtension();

  Optional<String> name();

  Optional<CodeableConcept> network();

  Optional<CodeableConcept> unit();

  Optional<Boolean> excluded();

  Optional<List<CoverageEligibilityResponse_Benefit>> benefit();

  Optional<String> id();

  Optional<List<CodeableConcept>> modifier();

  Optional<CodeableConcept> productOrService();

  Optional<List<CodeableConcept>> authorizationSupporting();

  Optional<uri> authorizationUrl();

  Optional<Reference> provider();

  static ImmutableCoverageEligibilityResponse_Item.Builder builder() {
    return ImmutableCoverageEligibilityResponse_Item.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/CoverageEligibilityResponse_Item.ts
import { CodeableConcept } from "./CodeableConcept";
import { CoverageEligibilityResponse_Benefit } from "./CoverageEligibilityResponse_Benefit";
import { Extension } from "./Extension";
import { Reference } from "./Reference";
import { uri } from "./uri";

export interface CoverageEligibilityResponse_Item {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    category: CodeableConcept | null | undefined;
    productOrService: CodeableConcept | null | undefined;
    modifier: CodeableConcept[] | null | undefined;
    provider: Reference | null | undefined;
    excluded: boolean | null | undefined;
    name: string | null | undefined;
    description: string | null | undefined;
    network: CodeableConcept | null | undefined;
    unit: CodeableConcept | null | undefined;
    term: CodeableConcept | null | undefined;
    benefit: CoverageEligibilityResponse_Benefit[] | null | undefined;
    authorizationRequired: boolean | null | undefined;
    authorizationSupporting: CodeableConcept[] | null | undefined;
    authorizationUrl: uri | null | undefined;
}
